### DISCUSSION TITLE
[Question]: Default SST + TTS Engines for all LibreChat instance users

### DISCUSSION THREAD

**Discussion by prononext** [danny-avila/LibreChat#3171] [Q&A]
### What is your question?

Is ther a way to set the default TTS and SST engines for all users of a instance to external?

### More Details

Having to go in the settings and  configure Browser to External for every user would be quite a hassle. 

### What is the main subject of your question?

UI, Endpoints

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There's an open PR for this: https://github.com/danny-avila/LibreChat/pull/2921

**Reply by prononext** [reply]
Very nice, even after intensive search I did not find it.

Some User Management with goups would be nice which prevent some groups from using expensive external SST and TTS.
